Formula/has.rb

```diff
@@ -0,0 +1,16 @@
+class Has < Formula
+  desc "Checks presence of various command-line tools and their versions on the path"
+  homepage "https://github.com/kdabir/has"
+  url "https://github.com/kdabir/has/archive/refs/tags/v1.4.0.tar.gz"
+  sha256 "0e73552dbf59e3da0d9254da87c94599595c9ea07c69a62a4853e69bbf3f0d7d"
+  license "MIT"
+  head "https://github.com/kdabir/has.git", branch: "master"
+
+  def install
+    bin.install "has"
+  end
+
+  test do
+    assert_match "has", shell_output("#{bin}/has has && false", 1)
```

Comment:
```suggestion
    assert_match "has #{version}", shell_output("#{bin}/has has")
```

Reply:
No, it will fail, because this version string has ansi escape code.

```shell
❯ has has|cat -v
^[[1m^[[38;5;2mM-bM-^\M-^T^[[m has ^[[1m^[[33;5;2m1.4.0^[[m
```

Reply:
Setting `$TERM` to "dumb" should be enough to disable that

Reply:
```shell
❯ TERM=dumb has has|cat -v
^[[1m^[[38;5;2mM-bM-^\M-^T^[[m has ^[[1m^[[33;5;2m1.4.0^[[m
```

Reply:
You should create an issue upstream for this, since that is not supposed to happen.

Reply:
I have created an issue for it before several months. But no response since now. <https://github.com/kdabir/has/issues/58>

Reply:
Now, just

```
❯ has
has v1.4.0
USAGE:    has <command-names>..
EXAMPLE:  has git curl node
```

Reply:
The latest version has supported `TERM=dumb`
![Screenshot from 2022-09-13 18-04-31](https://user-images.githubusercontent.com/32936898/189873834-ef933735-a365-4fb8-9c04-5ef69c2bc64c.png)